foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/DataLineageMigrationSteps.java

```diff
@@ -0,0 +1,51 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+public class DataLineageMigrationSteps extends AbstractMigrationTest {
+    private AbstractAissembleMigration migration;
+
+    @Given("an existing downstream project with data lineage is less than VERSION_AISSEMBLE_one_seven_zero")
+    public void anExistingDownstreamProjectWithDataLineageIsLessThanVERSION_AISSEMBLE_one_seven_zero() {
+        testFile = getTestFile("v1_7_0/DataLineageMigration/migration/CdiContainerFactory_1_6_0.java");
+        migration = new DataLineageAiopsReferencesMigration();
```

Comment:
A: We really should be constructing this in the `when` and making it a local variable. There's nothing useful we need it for, so creating a field that gets set in a multitude of `given`s just increases complexity without adding anything.